src/BuildingBlocks/Caching/Masa.BuildingBlocks.Caching/ICacheClient.cs

```diff
@@ -1,11 +1,11 @@
-﻿// Copyright (c) MASA Stack All rights reserved.
+// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
 namespace Masa.BuildingBlocks.Caching;
 
-public interface ICacheClient
+public interface ICacheClient : IDisposable
```

Comment:
ICacheClient should not support being released, it is a singleton, its life cycle is managed by the system, but ICacheClient created by the factory supports being released